### ISSUE TITLE
Research and document other network throttling profiles

### ISSUE THREAD

**Issue by phuedx** [phuedx/micro-device-lab#5]
- [Latency and Jitter in Mobile Networks](http://chimera.labs.oreilly.com/books/1230000000545/ch07.html#MOBILE_JITTER) from [Ilya Grigork's High Performance Browser Networking](http://shop.oreilly.com/product/0636920028048.do).
- [WebPageTest's `connectivity.ini.sample`](https://github.com/WPO-Foundation/webpagetest/blob/master/www/settings/connectivity.ini.sample)

**Comment by phuedx** [comment]
Ofcom published results of their 2014 and 2015 (late Q4, 2014) studies into 4G and 3G bandwidth and latency in the UK [here](http://stakeholders.ofcom.org.uk/market-data-research/other/telecoms-research/broadband-speeds/mobile-bb-nov14) and [here](http://stakeholders.ofcom.org.uk/market-data-research/other/telecoms-research/broadband-speeds/mobile-bb-april-15/) respectively.

**Comment by phuedx** [comment]
Google Chrome/Chromium's network throttling profiles now have download and upload bandwidth configured separately, thus µDL should configure download and upload bandwidth separately too.

**Comment by phuedx** [comment]
I'll track implementing the new network profiles in a separate ticket since, in practice, it's blocked on #7.

**Comment by phuedx** [comment]
Here's an example `profiles.yaml` file from the [Measuring mobile broadband performance in the UK (02/04/15)](http://stakeholders.ofcom.org.uk/market-data-research/other/telecoms-research/broadband-speeds/mobile-bb-april-15/) report:

``` yaml
profiles:
  - name: 4G (UK)
    bandwidth: # (Mbit/s)
      download: 14.7
      upload: 13.6
  - round_trip_time: 53.1 # (ms)

  - name: 3G (UK)
    bandwidth:
      download: 5.9
      upload: 1.6
    round_trip_time: 63.5
```
